development/docker-compose.dev.yml

```diff
@@ -19,3 +19,11 @@ services:
   mkdocs:
     volumes:
       - ../:/source
+  ui_build:
+    image: "local/nautobot-dev:local-${NAUTOBOT_VER}-py${PYTHON_VER}"
+    entrypoint: "bash -c 'cd /source/nautobot/project-static && npm run build -- --watch'"
```

Comment:
Is the idea that this will keep running and will auto-recompile on any change to the source files?

Reply:
Correct.

Reply:
> Also also also, I updated the local dev Docker Compose environment to run npm run build in watch mode to auto-recompile with webpack.